labellab_mobile/lib/screen/project/project_search_screen.dart

```diff
@@ -0,0 +1,163 @@
+              state.error != null
+                  ? ListTile(
+                      title: Text(state.error),
+                    )
+                  : Container(),
+              state.projects != null
+                  ? Column(
+                      children: state.projects
+                          .where((project) => project.name
+                              .toLowerCase()
+                              .contains(query.toLowerCase()))
+                          .map((project) => ProjectItem(
+                                project,
+                                onItemTapped: () {
+                                  _gotoProjectDetail(context, project.id);
+                                },
+                                onEditSelected: () {
+                                  _gotoEditProject(context, project.id);
+                                },
+                                onDeleteSelected: () {
+                                  _showDeleteConfirmation(context, project);
+                                },
+                              ))
+                          .toList(),
+                    )
+                  : Container(),
+            ],
+          )
+        : EmptyPlaceholder(description: "Your projects will appear here");
+  }
+
+  @override
+  Widget buildSuggestions(BuildContext context) {
+    return state.projects != null && state.projects.isNotEmpty
+        ? ListView(
+            children: <Widget>[
+              state.isLoading
+                  ? LinearProgressIndicator(
+                      backgroundColor: Theme.of(context).canvasColor,
+                    )
+                  : Container(
+                      height: 6,
+                    ),
+              state.error != null
+                  ? ListTile(
+                      title: Text(state.error),
+                    )
+                  : Container(),
+              state.projects != null
+                  ? Column(
+                      children: state.projects
+                          .where((project) =>
+                              project.name
+                                  .toLowerCase()
+                                  .contains(query.toLowerCase()) ||
+                              project.description
+                                  .toLowerCase()
+                                  .contains(query.toLowerCase()))
+                          .map((project) => ProjectItem(
+                                project,
+                                onItemTapped: () {
+                                  _gotoProjectDetail(context, project.id);
+                                },
+                                onEditSelected: () {
+                                  _gotoEditProject(context, project.id);
+                                },
+                                onDeleteSelected: () {
+                                  _showDeleteConfirmation(context, project);
+                                },
```

Comment:
Also, it's better to not have edit and delete options in the suggestions. It would be great if you can remove those options.

Reply:
Will do it